pkg/serving/config_changes.go

```diff
@@ -48,6 +48,21 @@ func UpdateEnvVars(config *servingv1alpha1.ConfigurationSpec, vars map[string]st
 
 }
 
+// ReplaceEnvVars clears any env vars configured in ConfigurationSpec and configures
+// the provided env vars
```

Comment:
I think the changes I'm suggesting make this file not really need to change at all.